test/functional/wallet_balance.py

```diff
@@ -57,14 +57,15 @@ def skip_test_if_missing_module(self):
         self.skip_if_no_wallet()
 
     def run_test(self):
-        self.nodes[0].importaddress(ADDRESS_WATCHONLY)
-        # Check that nodes don't own any UTXOs
-        assert_equal(len(self.nodes[0].listunspent()), 0)
-        assert_equal(len(self.nodes[1].listunspent()), 0)
+        if not self.options.descriptors:
```

Comment:
In commit "Move import and watchonly tests to be legacy wallet only in wallet_balance.py" (8ee94451ad55b81a53c38c6b8cee0c13f1b1f9e7)

It would be helpful to add the comment here from the commit description: "Imports and watchonly behavior are legacy wallet only, so make them only run when the test is in legacy wallet mode." It would also be helpful to say whether it is a good thing to be skipping these checks, or if the test maybe should be port these checks to work with a watchonly descriptor wallet in the future

Reply:
Added a comment. It is not necessary to test these for descriptor wallets, that's why they are disabled in the first place. Otherwise there would be a TODO.